orange/src/main/java/com/hartwig/hmftools/orange/algo/purple/PurpleVariantContextLoader.java

```diff
@@ -179,6 +186,15 @@ private static AllelicDepth extractRnaDepth(VariantContext context, @Nullable St
                 .orElse(null);
     }
 
+    @NotNull
+    private static List<String> extractReportableTranscripts(VariantContext context)
```

Comment:
Another minor but we typically use nullable annotations for every function input and output in ORANGE, so `context` should be annotated as `@NotNull`